caprover/one-click-apps/v4/apps/superset-only.yml

```diff
@@ -92,6 +92,10 @@ caproverOneClickApp:
       label: Postgres user:password
       validRegex: /:/
       description: user:password, copy the values from /#/apps/details/postgres
+    - id: '$$cap_admin_email'
+      label: Admin Email
+      defaultValue: 'admin@guardianconnector.net'
```

Comment:
I might not put this defaultValue here.  Having it in stack.example.yaml is enough, and is the place where "generic stack meets CMI hosting it for you on guardianconnector.net"